packages/flutter_tools/lib/src/commands/build_flx.dart

```diff
@@ -23,6 +23,7 @@ class BuildFlxCommand extends FlutterCommand {
     argParser.addOption('snapshot', defaultsTo: defaultSnapshotPath);
     argParser.addOption('depfile', defaultsTo: defaultDepfilePath);
     argParser.addOption('working-dir', defaultsTo: defaultWorkingDirPath);
+    argParser.addFlag('include-roboto-fonts', defaultsTo: false);
```

Comment:
I would have expected this to default to true and for the APK building to override this to false.